### DISCUSSION TITLE
When watching a repository, can I opt out of "Discussions"?

### DISCUSSION THREAD

**Discussion by mfn** [community/community#23790] [New to GitHub]
<p>I might be curious about a repositories technical side (issues/PRs) but not care much about the discussions, I couldn’t find a way to opt-out easily.</p>
<p>Is it possible?</p>
<p>I figured at least the email notifications contain a header which identifies them</p>
<pre><code class="lang-auto">Message-ID: &lt;repo/repo/repo-discussions/987654321@github.com&gt;
</code></pre>
<p>But that’s tedious having to set up a filter for each repo I would want that.</p>
<p>thanks</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi <a class="mention" href="/mfn">@mfn</a>! :wave:</p>
<p>Unfortunately at the moment we only have the ability to watch everything in the repository, or just watch for releases. We don’t yet offer the kind of granularity of notifications you’re looking for.</p>
<p>Filtering the emails is the current way to go - we have a help article here that will let you construct quite precise filters (although there’s nothing we can do about them being tedious to set up, sorry!)</p>
<p><a href="https://help.github.com/en/github/managing-subscriptions-and-notifications-on-github/configuring-notifications#filtering-email-notifications" class="onebox" target="_blank">https://help.github.com/en/github/managing-subscriptions-and-notifications-on-github/configuring-notifications#filtering-email-notifications</a></p>
<p>We’re always working to improve GitHub and the notifications system is getting a lot of work done on it at the moment, so you might want to submit a specific feature request for how you’d like it to work?</p>
<p>If you would, you can <a href="https://support.github.com/contact/feedback">submit it through our official product feedback form</a> and our product team will receive your request directly.</p>

**Comment by mfn** [comment]
<p>Thanks, filed a feature request :slight_smile:</p>
